docs/source/interfaces/Multi-dataset Fitting.rst

```diff
@@ -0,0 +1,138 @@
+the start and end `x` values are for this particular spectrum for each fit. The
+start and end values are editable and can be changed by the user by double
+clicking on an entry.
+
+.. figure::  ../images/multidataset-fitting/multidataset-spectra.png
+   :align: center
+   :width: 800px
+
+   The list of spectra after loading a workspace.
+
+Fit Property Browser
+--------------------
+
+The fit property browser is located on the left of the interface. The operation
+of the fit property browser is very similar to the `general fitting view
+<https://www.mantidproject.org/MBC_Setup_And_Execute_First_Fit>`_ available in
+Mantid but with slightly reduced functionality. The fit property browser is
+split into two major parts. Th top panel allows the user to define the
+specifics of the model they wish to fit to the dataset. The bottom panel
+contains the parameters for the minimizer and general parameters not associated
+with the model (e.g. the number of iterations). The bottom panel also allows
+the user to choose between simultaneous and sequential fitting mode:
+
+- **Simultaneous Fitting**: Fit all spectra at the same time as a multi-domain
+  function.
+
+- **Sequential Fitting**: Fit the spectra one after another, using the
+  parameters of the starting parameters of the first fit as input to the rest.
+
+Another import general option in the bottom panel is the *CreateOutput* option
+(available in sequential mode). With this checked, as well as the output
+parameters, the fit will also output a workspace with the normalised covariance
+matrices and workspace with parameters for each spectrum. It will also output
+one workspace for each spectrum containing the data, fitted model, and
+difference between model and data.
+
+.. figure::  ../images/multidataset-fitting/multidataset-fit-prop.png
+   :align: center
+   :width: 300px
+
+   The fit property browser. The top panel contains the specifics of the model
+   to fit. The bottom panel contains general parameters not associated with
+   the model.
+
+
+A model can be created by right clicking and choosing "Add function" in the top
+panel of the fit property window. This will produce a dialog box containing a
+large number of different fit functions which can be added to the model.
+
+.. figure::  ../images/multidataset-fitting/multidataset-fit-prop-dialog.png
+   :align: center
+   :width: 400px
+
+   The dialog containing the list of fit functions that can be added to the
+   model.
+
+After adding a function a new entry will appear in the top left panel with the
+name of the fit function and a list of fitting parameters. The user can enter
+appropriate guesses at starting values for the model parameters. Once a model
+has been created the "Plot Guess" checkbox can be used to show a plot of the
+model in the spectrum viewer using the starting parameter. To fit the model
+click the "Fit" button.
+
+
+.. figure::  ../images/multidataset-fitting/multidataset-fit-prop-dialog.png
+   :align: center
```

Comment:
It is the same picture as above. Maybe it should be an image showing the newly added function?